README.md

```diff
@@ -287,103 +264,163 @@ in your package and update the path definitions accordingly. Specifications are
-│   └── contributing.md
-├── hooks
-│   └── pre-commit
-├── repo
-│   ├── meta
-│   │   ├── schema
-│   │   │   ├── command-schema.json
-│   │   │   ├── config-schema.json
-│   │   │   ├── resource-schema.json
-│   │   │   └── package-schema.json
-│   │   └── version.json
-│   └── packages
-│       ├── B
-│       │   ├── bar
-│       │   │   ├── 0
-│       │   │   │   ├── command.json
-│       │   │   │   ├── config.json
-│       │   │   │   ├── marathon.json.mustache
-│       │   │   │   ├── resource.json
-│       │   │   │   └── package.json
-│       │   │   └── ...
-│       │   └── ...
-│       ├── F
-│       │   ├── foo
-│       │   │   ├── 0
-│       │   │   │   ├── config.json
-│       │   │   │   ├── marathon.json.mustache
-│       │   │   │   ├── resource.json
-│       │   │   │   └── package.json
-│       │   │   └── ...
-│       │   └── ...
-│       └── ...
-└── scripts
-    ├── 1-validate-packages.sh
-    ├── build.sh
-    └── install-git-hooks.sh
-```
+  ```bash
+  sudo pip install jsonschema
+  ```
 
-## Sources and Transfer Protocols
+3. Run the verification and build script:
 
-This section describes transfer of package metadata from a universe source to a client program.
+  ```bash
+  scripts/build.sh
+  ```
 
+4. Verify all build steps completed successfully
+5. Submit a pull request against the `version-3.x` branch with your changes. Every pull request opened will have a set
+   of automated verifications ran against it. These automated verification are reported against the pull request using
+   the GitHub status API. All verifications must pass in order for a pull request to be eligible for merge.
+
+6. Respond to manual review feedback provided by the DC/OS Community.
+  * Each Pull Request to Universe will also be manually reviewed by a member of the DC/OS Community. To ensure your
+    package is able to be made available to users as quickly as possible be sure to respond to the feedback provided.
+
+
+## Repository Consumption
+
+In order for Universe to be consumed by DC/OS the build process need to be ran to create the Universe Server.
+
+### Universe Server
+
+Universe Server is a new component introduce along side `packagingVersion` `3.0`. In order for Universe to be able to
+provide packages for many version of DC/OS at the same time, it is necessary for a server to be responsible for serving
+the correct set of packages to a cluster based on the clusters version.
+
+All Pull Requests opened for Universe and the `version-3.x` branch will have their docker image build and published
+to the DockerHub image [`mesosphere/universe-server`](https://hub.docker.com/r/mesosphere/universe-server/).
+In the artifacts tab of the build results you can find `docker/server/marathon.json` which can be used to run the
+Universe Server for testing in your DC/OS cluster.  For each Pull Request, click the details link of the "Universe Server
+Docker image" status report to view the build results.
+
+#### Build Universe Server locally
```

Comment:
There are a few occurrences of "docker" in this section that probably should be capitalized.